qcelemental/models/basis.py

```diff
@@ -145,7 +145,12 @@ class BasisSet(ProtoModel):
     @validator("atom_map")
     def _check_atom_map(cls, v, values):
         sv = set(v)
-        missing = sv - values["center_data"].keys()
+
+        # Center_data validation error, skipping
+        try:
+            missing = sv - values["center_data"].keys()
```

Comment:
if there was a pydantic validator on `center_data`, would that do the same trick (since first)?

Reply:
I do not believe so since pydantic first collects all validation errors then generates the error message. If a validator throws, it interrupts the collection process and throws an incorrect error.